core/metrics-core-service/src/main/java/org/hawkular/metrics/core/service/tags/SimpleTagQueryParser.java

```diff
@@ -199,13 +216,37 @@ public static boolean isRegExp(String tagValuesQuery) {
         Map<Long, List<Query>> costSortedMap =
                 QueryOptimizer.reOrderTagsQuery(tagsQueries, enableACostQueries);
 
+        List<Query> openshiftQuery = costSortedMap.get(QueryOptimizer.OPENSHIFT_OPTIMIZED_TAG_QUERY);
```

Comment:
This should be:
```
costSortedMap.get(QueryOptimizer.GROUP_OPENSHIFT_OPTIMIZATION)
```

Reply:
Ah, now the devil of autocorrection hits even the IDEs ;)